src/main/java/com/ernie/TicketApp/repository/TicketRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.ernie.TicketApp.repository;
+
+import com.ernie.TicketApp.model.Ticket;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+@Repository
+public interface TicketRepository extends JpaRepository<Ticket, UUID> {
+
```

Comment:
add at least 1 method for `TicketRepository` or `UserRepository` with custom query